packages/governance-sdk/src/governance/accounts.ts

```diff
@@ -946,12 +946,13 @@ export class Proposal {
     return this.isPreVotingState()
       ? governance.config.maxVotingTime
       : (this.votingAt?.toNumber() ?? 0) +
-          governance.config.maxVotingTime -
+          governance.config.maxVotingTime +
+          governance.config.votingCoolOffTime -
```

Comment:
`votingCoolOffTime` should be applied in both cases.
A better way to do this would be to rename `maxVotingTime` to `baseVotingTime` and then have a function for `maxVotingTime` -> `baseVotingTime` + `votingCoolOffTime`